informant.gemspec

```diff
@@ -11,4 +11,6 @@ Gem::Specification.new do |s|
   s.description = "Informant is a full-featured form builder for Ruby on Rails which promotes a simple syntax that keeps your views clean. Everything about a field (label, description, error display, etc) is encapsulated in a single method call."
   s.files       = `git ls-files`.split("\n") - %w[informant.gemspec Gemfile]
   s.require_paths = ["lib"]
+  s.add_dependency('actionview', '>= 4.0')
```

Comment:
That's right. I was thinking that you could change the major version number and we stick o semantic versioning that would mean the change in compatibility. Alternatively I could make it more 'elastic' and support Rails 3.x as well.